src/test/java/com/artipie/vertx/VertxSliceServerTest.java

```diff
@@ -212,6 +214,16 @@ public void serverMayStartOnRandomPort() {
         MatcherAssert.assertThat(srv.start(), new IsNot<>(new IsEqual<>(0)));
     }
 
+    @Test
+    void repeatedServerStartTest() {
+        this.start(
+            (s, iterable, publisher) -> {
+                throw new IllegalStateException("Request serving is not expected in this test");
+            }
+        );
+        Assertions.assertThrows(IllegalStateException.class, this.server::start);
```

Comment:
@pavlukhin check exception message

Reply:
I added a check in a simplest way. Extracting a constant might be a cleaner way to do it. Is it common in this codebase?